services/user.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = "",
+    last_name: str = "",
+) -> QuerySet[User]:
```

Comment:
The return type of `create_user` should be `User` instead of `QuerySet[User]` since it returns a single user instance.